### PR TITLE
Get platform from ctx

### PR DESCRIPTION
**Pull request by gustavorosolem**

### CODE REVIEW

File: node/middlewares/canonical.ts [hunk 1]
```diff
@@ -64,10 +63,10 @@ const routeFromCatalogPageType = (
 }
 
 export async function getCanonical (ctx: Context) {
-  const {clients: {canonicals, catalog, logger}, query: {canonicalPath}} = ctx
+  const {clients: {canonicals, catalog, logger}, query: {canonicalPath}, vtex: { platform }} = ctx
   const path = removeQuerystring(canonicalPath)
   let maybeRoute = await canonicals.load(path)
-  if (!Functions.isGoCommerceAcc(ctx)) {
+  if (platform !== 'gocommerce') {
```
Comment (kevinch) [inline node/middlewares/canonical.ts]: This should be a const.

File: node/package.json [hunk 2]
```diff
@@ -18,7 +17,7 @@
     "@types/node": "^7.0.8",
     "@types/ramda": "^0.26.8",
     "@types/route-parser": "^0.1.2",
-    "@vtex/api": "^3.24.1",
+    "@vtex/api": "^3.50.2",
     "tslint": "^5.11.0",
     "tslint-config-vtex": "^2.1.0",
     "typescript": "3.5.2"
```
Comment (kevinch) [inline node/package.json]: Missing changelog entry.

Comment (gustavorosolem) [reply node/package.json]: This project don't have changelog, I will not add this for now.

Comment (kevinch) [reply node/package.json]: Any time is time to add a changelog imo but ok.

**Pull request merged**